Resources/Prototypes/Body/Organs/arachnid.yml

```diff
@@ -1,42 +1,17 @@
 - type: entity
   id: BaseArachnidOrgan
-  parent: BaseItem
   abstract: true
+  suffix: arachnid
   components:
   - type: Sprite
     sprite: Mobs/Species/Arachnid/organs.rsi
-  - type: Organ
-  - type: Food
-  - type: Extractable
-    grindableSolutionName: organ
-  - type: SolutionContainerManager
-    solutions:
-      organ:
-        maxVol: 10
-        reagents:
-        - ReagentId: Nutriment
-          Quantity: 10
-      food:
-        maxVol: 5
-        reagents:
-        - ReagentId: UncookedAnimalProteins
-          Quantity: 5
-  - type: Tag
-    tags:
-      - Meat
 
 - type: entity
+  parent: [OrganAnimalStomach, BaseArachnidOrgan]
```

Comment:
All inline lists should have spaces inside the brackets, like you did with the previous ones